src/App.tsx

```diff
@@ -30,7 +31,10 @@ function App() {
           </StrictMode>
           <Routes>
             <Route path="/">
-              <Route index element={<HomePage />} />
+              <Route
+                index
+                element={isAuthenticated() ? <HomePage /> : <SettingPage />}
```

Comment:
Redirection to HomePage doesn't seem to always work.

![Sep-26-2024 00-22-44](https://github.com/user-attachments/assets/fd96f4c2-5130-4df0-b16b-2532eec2b6dd)

Reply:
This issue is till observed.